tests/test_virtual_chassis.py

```diff
@@ -0,0 +1,71 @@
+import time
+import json
+import redis
+import pytest
+from swsscommon import swsscommon
+
+class TestVirtualChassis(object):
+    def check_chassis_sysport(self, dvs):
+        # check if instance is connected to a chassis db
+        cfgFile = "/usr/share/sonic/virtual_chassis/default_config.json"
+        (ret, out) = dvs.runcmd("sonic-cfggen -v  DEVICE_METADATA.localhost.chassis_db_address -j %s" % cfgFile)
+        verifyChassDb = True
+        if ret != 0 or out.strip() == "":
+            # instance has no connection to chassis db
+            verifyChassDb = False
+            print "verify vs without chassisDb"
```

Comment:
Use `print()` rather than `print` now that we are using `python3`.

Reply:
ack